### ISSUE TITLE
Example code does not compile.

### ISSUE THREAD

**Issue by joboscribe** [rust-lang/www.rust-lang.org#651]
# Technical Bug Report
## Your Environment
| Software         | Version(s) |
| Rust | 1.30.1 |
| Form Factor      | Desktop |
| Browser Type     | Chrome |
| Browser Version  | 70.0.3538.110 |

## Is this an issue with Rust, CSS, or HTML (or a combination)?
I'm going to say HTML.

## Expected Behavior
If i correctly followed (or copy/pasted) the code example under the heading "A Small Rust Application" then i should be able to compile and run the sample program.

## Current Behavior
Compiling the sample code results in: 
```
error[E0432]: unresolved import `ferris_says`
 --> src/main.rs:1:5
  |
1 | use ferris_says::say; // from the previous step
  |     ^^^^^^^^^^^ Maybe a missing `extern crate ferris_says;`?

error: aborting due to previous error
```

## Step to Reproduce
Follow the instructions on the /learn/get-started page through the "A Small Rust Application" section.

**Comment by joboscribe** [comment]
My point was that a new user, following the instructions on the page, will be left with code that does not compile. That seems like a poor initial experience for potential rustaceans.

**Comment by sh4338** [comment]
That makes more sense.  Sorry.  And I completely agree.

**Comment by mbrubeck** [comment]
> My point was that a new user, following the instructions on the page, will be left with code that does not compile. That seems like a poor initial experience for potential rustaceans.

It wlll work if they are using the current stable version of Rust.

**Comment by joboscribe** [comment]
I went through the example again and it does indeed now compile and run just fine. Thanks.

**Comment by xgqfrms** [comment]
> 1. Cargo.toml

where is the path of `Cargo.toml`?

**Comment by xMxster** [comment]
You totally should update the example in the website. Just starting and stuff being broken when you're literally copy-pasting the example gave to you by the starting guide isn't cool at all

**Comment by xMxster** [comment]
Okay, my bad, my bad completely sorry, I won't even edit the prior comment to **show how dumb, we developers can be, even with instructions crystal clear.**

**I was adding `ferris-says = 0.2` at the bottom of `[package]`, and not under `[dependencies]`**

I'll leave these comments, so others know it's okay to be wrong, admit it, apologize, and then let your experience help others, even if you make yourself look very dumb.

**Comment by wanghs-ctrl** [comment]
> 这是由于您的 Rust 版本所致。 Rust 1.31 引入了一个称为“版本”的新概念，详细解释请参见[此处](https://doc.rust-lang.org/edition-guide/editions/index.html)。其基本要点是 Rust 2018 版引入了许多 Rust 2015 版中没有的重大更改。由于 Rust 2018 是完全可选的，并且是 Rust 1.31 中的一项新功能，因此使用 Rust 1.30.x 及更早版本创建的任何 crate 将默认为 Rust 2015。在 Rust 2015 中，`extern crate ferris_sys`需要`main.rs`;在 Rust 2018 中，事实并非如此。 Rust 版本是通过在标题下`Cargo.toml`插入来指定的。`edition = “year”``[package]`
> 
> TLDR：解决问题的两种方法：
> 
> 1. 将 Rust 更新到 1.31.x 或更高版本，并插入`edition = “2018”`到以下行`authors`下`Cargo.toml`
> 2. 插入`extern crate ferris_sys`到顶部`main.rs`
> 
> 学习/入门页面底部的一些简明解释可能会很有用。

cargo 1.71.0 (cfd3bbd8f 2023-06-08)


>

**Comment by wanghs-ctrl** [comment]
cargo 1.71.0 (cfd3bbd8f 2023-06-08)
Cargo.toml --> edition = "2021"
依然还是出错：
error[E0432]: unresolved import `ferris_says`
 --> .\main.rs:1:5
  |
1 | use ferris_says::say; // from the previous step
  |     ^^^^^^^^^^^ maybe a missing crate `ferris_says`?
  |
  = help: consider adding `extern crate ferris_says` to use the `ferris_says` crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.

**Comment by wanghs-ctrl** [comment]
cargo 1.71.0 (cfd3bbd8f 2023-06-08)
Cargo.toml --> edition = "2021"

仍然出错：
error[E0432]: unresolved import ferris_says
--> .\main.rs:1:5
|
1 |使用 ferris_says::say; // 从上一步开始
| ^^^^^^^^^^^^ 也许缺少一个箱子ferris_says？
|
=帮助：考虑添加extern crate ferris_says使用ferris_says板条箱

错误：由于先前的错误而中止

有关此错误的更多信息，请尝试rustc --explain E0432。

**Comment by senekor** [comment]
@wanghs-ctrl I can't read this, but "unresolved import" makes me guess you forgot to add the dependency in `Cargo.toml`:
```
[dependencies]
ferris-says = "0.2"
```

**Comment by wanghs-ctrl** [comment]
@senekor 不是原因，ferris-says = "0.2" 已经添加过了

**Comment by senekor** [comment]
I don't speak Chinese. I'd have to see the code to be able to help.
